packages/api/schema/stryker-core.json

```diff
@@ -398,6 +398,11 @@
       "type": "number",
       "default": 5000
     },
+    "initialTimeoutMIN": {
```

Comment:
Can this be renamed to `dryRunTimeoutMinutes`?

Reasoning:

* "initial timeout" can be confused with other "timeoutxxx" options.
* "Min" can be confused with "minimum" or something like that.